scantool/scantool_850.c

```diff
@@ -95,13 +95,454 @@ struct dtc_table_entry {
+	//  Define all known, interpreted ECU 7A (Motronic 4.4) DTCs.
+	//----------------------------------------------------------
+	// ECU 7A has 93 DTCs: 0A 0C 0D 0E 0F 10 11 13 18 1A 1B 1D 28 31 32 33 34 35 36 37 39 3D 3E 3F 40 41 42 43 44 4B 4D 50 51 53 54 55 56 59 5D 5E 5F 60 61 62 63 65 67 6B 6D 6E 6F 70 73 75 77 78 79 7A 7B 7C 7F 80 81 82 96 97 98 99 9A 9B A4 A8 A9 AA B4 B5 B6 B7 B8 B9 D2 D3 DE E1 E5 E6 ED EE EF F1 F4 FD FE
+	{0x7A, 0x0A, 212, "HO2S sensor signal, front"},
+	{0x7A, 0x0C, 153, "HO2S sensor signal, rear [P0136 (faulty), P0137 (low), or P0138 (high)]"},
+	{0x7A, 0x0D, 521, "Front HO2S, preheating"},
+	{0x7A, 0x0E, 522, "Rear HO2S, preheating malfunction [P0141]"},
+	{0x7A, 0x0F, 435, "Front HO2S slow response"},
+	{0x7A, 0x10, 436, "Rear HO2S compensation [P0133] [see 1st 3 items in Bing search for 'P0133 \"EFI-436\"' and 1st 3 items in Google search]"},
+	{0x7A, 0x11, 425, "Rear HO2S, control (or possibly Temperature warning, level 1)"},
+	{0x7A, 0x13, 343, "Fuel Pump Relay"},
+	{0x7A, 0x18, 211, "CO Potentiometer"},
+	{0x7A, 0x1A, 231, "Long term fuel trim, part load [P0171 or P0172] [defective MAF, fuel pressure high/low (P0171/P0172), elbow at intake manifold (hidden underneath coolant thermostat) or elbow at PTC or vacuum leaks (if P0172), see OTP Volvo 850 DVD, see 7-step checkup in section including \"2-3-1 (Adaptive HO2S Control Lean At Part Load)\" of \"http://www.volvotips.com/index.php/850-2/volvo-850-s70-v70-c70-service-repair-manual/volvo-850-diagnostics-with-codes-obd/\"]"},
+	{0x7A, 0x1B, 232, "Long term fuel trim, idling [P0172 or P0171] [elbow at intake manifold (hidden underneath coolant thermostat) or elbow at PTC or vacuum leaks (if P0172), fuel pressure high/low (P0171/P0172), defective MAF, see OTP Volvo 850 DVD, see 10-step checkup in section \"2-3-2 (Adaptive HO2S Control Provides Richer Mixture At Idle)\" of \"http://www.volvotips.com/index.php/850-2/volvo-850-s70-v70-c70-service-repair-manual/volvo-850-diagnostics-with-codes-obd/\"]"},
+	{0x7A, 0x1D, 666, "DTC in transmission CM"},
+	{0x7A, 0x28, 443, "TWC efficiency"},
+	{0x7A, 0x31, 542, "Misfire, more than 1 cylinder"},
+	{0x7A, 0x32, 451, "Misfire cylinder 1"},
+	{0x7A, 0x33, 452, "Misfire cylinder 2"},
+	{0x7A, 0x34, 453, "Misfire cylinder 3 [P0303]"},
+	{0x7A, 0x35, 454, "Misfire cylinder 4"},
+	{0x7A, 0x36, 455, "Misfire cylinder 5"},
+    ////{0x7A, 0x37, 445, "??Misfire cylinder 6??.  [There is another more believable code that maps to EFI-445, so let's leave this commented out.]"},
+	{0x7A, 0x39, 353, "Immobilizer"},
+	{0x7A, 0x3D, 544, "Misfire, more than 1 cylinder"},
+	{0x7A, 0x3E, 543, "Misfire exhaust value of at least one cylinder"},
+	{0x7A, 0x3F, 551, "Misfire, cylinder 1"},
+	{0x7A, 0x40, 552, "Misfire, cylinder 2"},
+	{0x7A, 0x41, 553, "Misfire, cylinder 3"},
+	{0x7A, 0x42, 554, "Misfire, cylinder 4"},
+	{0x7A, 0x43, 555, "Misfire, cylinder 5"},
+	{0x7A, 0x44, 556, "Misfire, cylinder 6"},
```

Comment:
You might be saying to yourself, "I've _never_ had ignition in cylinder 6, so shouldn't this DTC always be set?" But the Motronic M4.4 (or M4.4.1) was also used in some model years of 960/S90/V90/C90. But then 0x37 should probably map to something like EFI-456 instead of EFI-445.